Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml

```diff
@@ -497,21 +497,6 @@
         - ReagentId: Fat
           Quantity: 3
 
-- type: entity
-  name: meatwheat clump
-  parent: FoodMeatBase
-  id: FoodMeatWheat
-  description: This doesn't look like meat, but your standards aren't that high to begin with.
-  components:
-  - type: Sprite
-    state: clump
-  - type: SolutionContainerManager
-    solutions:
-      food:
-        reagents:
-        - ReagentId: UncookedAnimalProteins
-          Quantity: 1
-
```

Comment:
This should be commented out instead.

Reply:
Did meat wheat get added to the normal files?

Reply:
Was it a floof change before? let me verify.

Reply:
This change is also visible in EE, this is upstream best is to comment it out to avoid conflicts.

Reply:
I changed it back. Thought I forgot I added meat wheat in the wrong place.